packages/medusa-plugin-slack-notification/package.json

```diff
@@ -43,5 +43,9 @@
     "medusa-test-utils": "^1.1.37",
     "moment": "^2.27.0"
   },
-  "gitHead": "c46300d58fbfd0b2dc2c02745ae143e6247e885b"
+  "gitHead": "c46300d58fbfd0b2dc2c02745ae143e6247e885b",
+  "keywords": [
+    "medusa-plugin",
+    "medusa-plugin-notification"
```

Comment:
todo: Not a notification plugin either, but again hard to know 😅 This is Slackbot forwarding confirmed orders to a channel

Reply:
I thought that categorized it as a plugin 😅 Again what should it be for this one?

Reply:
I think `medusa-plugin` should suffice. Similar to the wishlist plugin

Reply:
For reference, a notification plugin will follow our notification-service interface. Same goes for payment, fulfillment, etc.

Reply:
On second thought, I'm fine with keeping them in the notification grouping for now, so feel free to merge!